### PR TITLE
Implemented experimental configs:

### PR DESCRIPTION
**Pull request by ijunaid**
- enableVisibilityTracking
- enablePreviousNameRecording

### CODE REVIEW

File: ios/Classes/CountlyFlutterPlugin.m [hunk 1]
```diff
@@ -1665,6 +1665,15 @@ - (void)populateConfig:(NSDictionary *)_config {
             config.indirectAttribution = attributionValues;
         }
 
+        NSNumber *visibilityTracking = _config[@"visibilityTracking"];
+        if (visibilityTracking) {
+            config.experimental.enableVisibiltyTracking = [visibilityTracking boolValue];
```
Comment (arifBurakDemiray) [inline ios/Classes/CountlyFlutterPlugin.m:1670]: typo here *enableVisibilityTracking

**Review by arifBurakDemiray — CHANGES_REQUESTED** [review]

File: ios/Classes/CountlyFlutterPlugin.m (hunk 1 above)
Comment (ijunaid) [reply ios/Classes/CountlyFlutterPlugin.m:1670]: but this typo is not fixable here :)
It was already released in iOS

Comment (arifBurakDemiray) [reply ios/Classes/CountlyFlutterPlugin.m:1670]: ohhh okey :/

Comment (turtledreams) [reply ios/Classes/CountlyFlutterPlugin.m:1670]: :D

**Review by arifBurakDemiray — APPROVED** [review]

**Review by turtledreams — APPROVED** [review]

**Pull request merged**